### DISCUSSION TITLE
TR and image_scaling options

### DISCUSSION THREAD

**Discussion by geowk** [CoBrALab/RABIES#290] [General]
Hello,

1. What does 'auto' mean for the TR option in preprocessing and confound correction. If one does not enter a TR value, does it mean that RABIES reads the TR information from the .json of the input datasets?  Is this a per subject or just one value across the entire input dataset? What if you have a input dataset of 20 rodents, 10 with a TR 1.0s and the other 10 with a TR of 1.5s and you don't specify a TR: What would 'auto' do in that scenario?

2. Image_Scaling: What is the difference between grand_mean scaling and voxelwise_mean? When it says "across voxel" for grand_mean scaling definition, does that imply the mean across all voxels and dividing each voxel's timeseries by that grand-mean timecourse?  When we choose the voxelwise_mean option, the voxelwise mean value is around 0 and not 100. Should it be around 100 or is the final result just multiplied by 100? We are using Rabies 0.4.8

Thanks.

g.

**Comment by Gab-D-G** [comment]
Hi, those are good points to clarify.

1. With 'auto', the TR is read from the Nifti file header, so you must make sure that this information is accurate. It is read from each scan independently, so if the TR varies across scans, it would adapt accordingly.
2. With grand mean scaling, all timeseries are divided by a single scalar: the mean signal across the entire 4D timeseries. With voxelwise mean, the scaling is conducted for each voxel independently, by dividing the voxel timeseries by its temporal mean. After applying either scaling, the multiplication by 100 only means that a value of 1 would be 1% deviation from the mean. You certainly would not expect 100, as BOLD fluctuations are generally only a few percent at maximum.

Hope this helps,
Gabe DG

**Comment by geowk** [comment]
Thanks for clarifying, Gabe.

g.

**Comment by Gab-D-G** [comment]
Anytime!
